src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1468,7 +1489,11 @@ default boolean isFromGuild()
      *             <li>{@link net.dv8tion.jda.api.Permission#MESSAGE_MANAGE Permission.MESSAGE_MANAGE}</li>
      *         </ul>
      * @throws java.lang.IllegalArgumentException
-     *         If the provided unicode emoji is null or empty.
+     *         If the provided unicode emoji is null or empty or if the provided user is null.
+     * @throws java.lang.IllegalStateException
+     *         If this message was <b>not</b> sent in a
+     *         {@link net.dv8tion.jda.api.entities.TextChannel TextChannel}
```

Comment:
I'd say not sent in a guild to also account for other channel types such as news channels

Reply:
Current behavior doesn't mean its permanent, being vague allows us to make changes more easily.

Reply:
How should I implement it then? As it is now or should I revert that back? Or do I get your comment wrong?